starfish/intensity_table/intensity_table.py

```diff
@@ -94,25 +86,26 @@ def _build_xarray_coords(
         return coordinates
 
     @classmethod
-    def empty_intensity_table(
+    def empty(
```

Comment:
same concerns w/ `.empty()` vs `.zeroes()` and expected behavior given numpy's API